guides/data.livemd

```diff
@@ -309,3 +309,97 @@ Vl.new(title: "Density heatmap", width: 400, height: 250)
   text: "sepal_width"
 )
 ```
+
+### Jointplot
+
+A jointplot shows the relationship between two variables and the distribution of individuals of each one. Expects quantitative data.
+
+```elixir
```

Comment:
I think the first example should be using Data.joint_plot. Maybe we should even stop showing the original VegaLite version? They are getting too long and if you want to see its representation, you can do IO.inspect, WDYT?

Reply:
Yep! I agree with you, I’d just say to rethink the guide before we release the module. At the moment it’s being more of a guide to myself!